app/konnect/updates.md

```diff
@@ -11,6 +11,9 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## April 2023
 
+**Allow filtering of runtime instances based on connection status** 
+: Runtime administrators can now filter runtime instances based on connection statuses `connected`, `disconnected`, `all`.
```

Comment:
```suggestion
: Runtime administrators can now filter runtime instances based on connection statuses `connected`, `disconnected`, and `all`.
```
Or would it be "or"? Not sure if you can filter based on multiple at the same time.